### DISCUSSION TITLE
Make the language cookie value extendable

### DISCUSSION THREAD

**Discussion by Xenossolitarius** [nuxt-modules/i18n#2907] [Q&A]
### Describe the feature

I am currently trying to plug in a Nuxt 3 based site into an old web ecosystem and I am using `prefix_except_default`  strategy. I am interested in keeping the old behavior to use short name prefixes like "en" or "es" but for the sake of supporting cross domain language cookie I would like to use "en-US" or "es-ES" when checking the cookie. Currently it's using the same value as the url prefix. If this is not in plan, can you give me some short guidance how to hack it? Probably intercepting the cookie on the server side?

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution) (The contribution guideline of nuxt-modules/i18n is compliant with Nuxt too).
- [X] Check existing [discussions](https://github.com/nuxt-modules/i18n/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by kazupon** [comment]
This is a question, not issue. so I'll move to github discussions.

**Reply by kazupon** [reply]
Could you be more specific about what you want to achieve using exmaple?
